robottelo/ui/base.py

```diff
@@ -163,14 +202,9 @@ def delete_entity(self, name, really, name_locator, del_locator,
         # Make sure that element is really removed from UI. It is necessary to
         # verify that fact few times as sometimes 1 second is not enough for
         # element to be actually deleted from DB
+        self.is_entity_delete = True
         for _ in range(3):
```

Comment:
Actually +1 to @abalakh's question.  Is there a way to ensure that self.is_entity_delete is reset at all times inspite of any kind of failures on delete.